src/main/java/net/dv8tion/jda/core/entities/EntityBuilder.java

```diff
@@ -797,10 +795,10 @@ public Message createMessage(JSONObject jsonObject, MessageChannel chan, boolean
                     }
                     break;
                 default:
-                    WebSocketClient.LOG.debug("Received an unknown activity type in a message. JSON: {}", activityData);
                     break;
-
             }
+
+            Checks.check(!(activityType != MessageActivity.ActivityType.LISTENING && application == null), "Either the ActivityType is wrong or the Application is null!");
```

Comment:
We shouldn't fail creating a message just because of this

Reply:
Thought about that. Should ask next time before i commit i guess. Can i DM you on discord?